docs/virtualenv.rst

```diff
@@ -19,7 +19,7 @@ are that you'll need more than one version of it, or the numerous other Python
 applications it depends upon, at any one time.
 
 For example, when a new version of Django is released, you might want to check
-your project is still compatible. You don't want to have to set up a whole new
+to see if your project is still compatible. You don't want to have to set up a whole new
```

Comment:
It's always a good idea to make commits as atomic (i.e. as small as possible, reduced to the minimum possible meaningful changes), and to place changes that address different things into different commits (as you have done).

However it's also a good idea to put them into separate pull requests (i.e. separate branches) so the project maintainer can accept one without accepting the other.